src/shaders/metallic-roughness.frag

```diff
@@ -341,43 +382,85 @@ void main()
     // Anything less than 2% is physically impossible and is instead considered to be shadowing. Compare to "Real-Time-Rendering" 4th editon on page 325.
     vec3 specularEnvironmentR90 = vec3(clamp(reflectance * 50.0, 0.0, 1.0));
 
+    // LIGHTING
+
+    vec3 color = vec3(0.0, 0.0, 0.0);
+    vec3 normal = getNormal();
+    vec3 view = normalize(u_Camera - v_Position);
+
     MaterialInfo materialInfo = MaterialInfo(
         perceptualRoughness,
         specularEnvironmentR0,
         alphaRoughness,
         diffuseColor,
         specularEnvironmentR90,
-        specularColor
+        specularColor,
+        normal,
+        0.0 //Clearcoat factor is null
+    );
+    MaterialInfo clearCoatInfo = MaterialInfo(
+        clearcoatRoughness,
+        vec3(0.04), //fixed from specification //todo use variable from functions
```

Comment:
what “functions” are meant by that `todo`?

Reply:
This todo reminds me to discuss the usage of fresnel appoximation with Nobert